db/version_builder.cc

```diff
@@ -1046,6 +1107,13 @@ class VersionBuilder::Rep {
         return s;
       }
     }
+
+    if (track_found_and_missing_files_ && version_updated) {
+      version_updated_since_last_check_ = true;
+      if (!edited_in_atomic_group_ && edit->IsInAtomicGroup()) {
+        edited_in_atomic_group_ = true;
```

Comment:
IIRC, Zorro sets atomic_flush to true, even though it uses only one column family. Maybe we should disable this feature only if # of column families > 1. Longer term, we might want to make this work for atomic flush also.

Reply:
Yes, we can follow up on this and figure a solution for atomic flush too since that could be ZippyDb's future needs.